### PR TITLE
support logging build output to JSON in order for Domino pipeline ingestion

### PR DESCRIPTION
**Pull request by steved**
WIP obviously, but this was my first pass and along with https://github.com/cerebrotech/charts/pull/552 it actually seems to work. Not sure I love the way it's formulated so it's worth discussion.

### CODE REVIEW

**Comment by sonnysideup** [comment]
@steved I know this is a WIP but just let me know when you want to put :eyes: on it.

File: internal/builder/embedded/util.go [hunk 1]
```diff
@@ -72,53 +72,60 @@ func solveRequestWithContext(sessionID string, image string, opts *config.BuildO
...
+				Timestamp: v.Timestamp,
+			})
 		}
+
+		for _, displayChannel := range displayChannels {
+			displayChannel <- &s
+		}
+	}
+
+	for _, displayChannel := range displayChannels {
```
Comment (steved) [inline internal/builder/embedded/util.go:116]: defer

File: controllers/start.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func StartManager(metricsAddr string, enableLeaderElection bool, brokerOpts *mes
 }
 
 func reexec() {
-	if len(os.Getenv("IMG_DO_UNSHARE")) <= 0 && system.GetParentNSeuid() != 0 {
+	if len(os.Getenv("IMG_RUNNING_TESTS")) <= 0 && len(os.Getenv("IMG_DO_UNSHARE")) <= 0 && system.GetParentNSeuid() != 0 {
```
Comment (steved) [inline controllers/start.go:86]: @sonnysideup how do we `delve` debugging locally once Forge does the `unshare`?

**Comment by steved** [comment]
@sonnysideup please review. Ignore all `Dockerfile`, `e2e`, `cmd/`, `config/`, and `skaffold` changes.

**Comment by sonnysideup** [comment]
replaced by #22

**Pull request closed without merging**